articles/implementing-network-address-translation/index.md

```diff
@@ -0,0 +1,205 @@
+If internal devices have been assigned all the available global addresses, any device requesting a public address to reach the internet will have to wait until one is made available.
+
+#### Port Address Translation (PAT).
+Dynamic NAT is use by an organizations to connect their devices, but some organizations need to have most if not all devices reach the internet.
+If their network is large, it requires a huge set of registered public addresses; thus, it completely defeats NAT's goal.
+Dynamic NAT lessens this problem to some degree. However, if a large percentage of internal hosts need access to the internet, Port Address Translation, also called NAT overload, solves this problem.
+To understand how PAT works, it is important to recall how the host uses the Transmission Control Protocol (TCP) and User Datagram Protocol (UDP) and port numbers to transmit data, which can be found here [Understanding TCP/IP Transport Layer Protocols: TCP and UDP](https://www.section.io/engineering-education/understanding-tcp-ip-transport-layer-protocols/).
+With these protocols, PAT can map multiple private addresses to one or a few public addresses by ensuring that devices use different TCP and UDP port numbers for each session with a server on the internet.
+![PAT](/engineering-education/implementing-network-address-traslation/pat.jpg)
+
+### NAT configuration.
+In the first half of this article, a brief introduction  on NAT and different types of NAT were explained.
+This second part will cover how to implement Static NAT, Dynamic NAT, and PAT on a CISCO router.
+
+####  Steps to Configuring Static NAT.
+Static NAT is configured using two steps, which are:
+1. Creating a mapping between the private internal address and public global address using the  **ip nat inside source static *private address public address*** global configuration command.
+2. After the mapping is made, the interfaces partaking in NAT translation are configured as either **inside** or **outside** comparative with NAT.
+The router interface associated with the LAN is assigned the inside interface using the **ip nat inside** interface mode command.
+The router interface associated with the internet is assigned the outside interface using  the **ip nat inside** interface  mode command
+
+![Static NAT topology](/engineering-education/implementing-network-address-traslation/statictopo.jpg)
+In the figure above, the Gigabit 0/0 (g0/0) interface is the inside interface because it is connected to the LAN. In contrast, the S0/0/0 interface is configured as the outside interface because it connects to the internet.
+### Configuring static NAT
+To configure a static NAT between the private address **172.31.1.2** and public address **200.100.100.2**:
+- Map the server's private address 172.31.1.2 to the public routable address 200.100.100.2 using the command:**Ip nat inside source static 172.31.1.2 200.100.100.2**
```

Comment:
When specifying commands, specify them within triple opening and closing back quotes like this
```
ip nat inside source static 172.31.1.2 200.100.100.2
```
Make the changes throughout the article for every commands.